### PR TITLE
#714 refactoring before extract validation

### PR DESCRIPTION
**Pull request by rozhkovdmitrii**

### CODE REVIEW

File: proxy/neon_proxy_app.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+from .proxy import entry_point
+from .mempool.mempool_service import MemPoolService
+from .statistics_exporter.prometheus_proxy_server import PrometheusProxyServer
+
+
+class NeonProxyApp:
+
+    def __init__(self):
+        self._mempool_service = MemPoolService(is_background=True)
```
Comment (afalaleev) [inline proxy/neon_proxy_app.py]: Remove this from the develop branch until it will be tested on live cluster.

File: proxy/neon_proxy_app.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+    def start(self):
+        PrometheusProxyServer()
+        self._mempool_service.start()
```
Comment (afalaleev) [inline proxy/neon_proxy_app.py]: Remove this from the develop branch until it will be tested on live cluster.

**Review by afalaleev — CHANGES_REQUESTED** [review]

File: proxy/neon_proxy_app.py (hunk 1 above)
Comment (rozhkovdmitrii) [reply proxy/neon_proxy_app.py]: Got rid of it

File: proxy/neon_proxy_app.py (hunk 2 above)
Comment (rozhkovdmitrii) [reply proxy/neon_proxy_app.py]: got rid of it

**Review by afalaleev — DISMISSED** [review]

**Review by afalaleev — APPROVED** [review]

**Pull request merged**